roles/ansible-test/defaults/main.yaml

```diff
@@ -14,6 +14,7 @@ ansible_test_docker: false
 # Default ansible-test options
 ansible_test_git_branch: 'devel'
 ansible_test_python: "{{ ansible_python_version.split('.')[:2]|join('.') }}"
+ansible_test_docker_python: false
```

Comment:
this shouldn't be needed, we can just remove it and set ansible_test_python with the change below